deployments/deploy-tests.ps1

```diff
@@ -2,16 +2,143 @@ try {
 
     #shows how to import modules
     Import-Module .\powerform\pf-deployment-context\pf-deployment-context.psd1
-    Import-Module .\azure\pf-azure-context\pf-azure-context.psd1
+    Import-Module .\azure\resource-containers\pf-subscription\pf-subscription.psd1
+    Import-Module .\azure\resource-containers\pf-resource-group\pf-resource-group.psd1
     Import-Module .\azure\resources\pf-key-vault\pf-key-vault.psd1
     Import-Module .\azure\resources\pf-log-analytics\pf-log-analytics.psd1
 
     #shows what functions are exported from each module
+    Get-Command -Module pf-subscription
     Get-Command -Module pf-key-vault
     Get-Command -Module pf-log-analytics
     Get-Command -Module pf-deployment-context 
     Get-Command -Module pf-azure-context
 
```

Comment:
I've added alot of "demos" on how to use all this together.  Anytime we add a new feature or resource, the tests are starting to get exponentially more dynamic on what you can do with them.  I am now finding we can do 4 or 5 different ways to accomplish the same task.  So if you want to understand everything I adding, this is probably the file to read through and get any questions from about how this module is going to be used and what it can handle/can't.